### DISCUSSION TITLE
Pop Up Abt us

### DISCUSSION THREAD

**Discussion by YesTheProxy** [theshmunguss1/RamOS#15] [Ideas]
Ok so this is mostly for @DJshelfmushroom because he's a coding genius is there any way you could make an about us that pops  
up. like a section for me one for @theshmunguss1 and one for @DJshelfmushroom

**Comment by DJshelfmushroom** [comment]
like a little widget? kinda like this is what im thinkin: https://jqueryui.com/dialog/

**Reply by YesTheProxy** [reply]
Exactly

**Reply by DJshelfmushroom** [reply]
i have someone working on that i think
